doc/modules/mod_revproxy.md

```diff
@@ -33,23 +33,36 @@ For more details about the listeners configuration please take a look at [ejabbe
 
 #### Configuring routes
 
-To define reverse proxy rules, add the following entry to the modules section.
+To define reverse proxy rules, add entries defining routes to `modules.mod_revproxy.routes`.
 
-```Erlang
-{mod_revproxy,
-   [{routes, [{"www.erlang-solutions.com", "/admin", "_",
-               "https://www.erlang-solutions.com/"},
-              {":var.com", "/:var", "_", "http://localhost:8080/"},
-              {":domain.com", "/", "_", "http://localhost:8080/:domain"}]
-     }]},
-```
+#### `modules.mod_revproxy.routes.host`
+* **Syntax:** non-empty string
+* **Default:** no default
+* **Example:** `host = "www.erlang-solutions.com"`
+
+#### `modules.mod_revproxy.routes.path`
+* **Syntax:** string
+* **Default:** no default
```

Comment:
```suggestion
* **Default:** `""`
```

Reply:
I don't think there is a default as if I'm not mistaken leaving this option empty wouldn't pass parsing.